include/verilated.mk.in

```diff
@@ -208,8 +208,8 @@ endif
 #   2. Create a new archive from extracted .o and given .o
 %.a:
 	$(info Archive $(AR) -rcs $@ $^)
-	$(file >$@.tmp)
```

Comment:
The "touch" replacement isn't fully equivalent to this line -- did you intend to truncate the file (e.g. as a safety measure), or was it just to ensure that the file would always be present? (It should not be necessary to truncate it from what I could see, but it would be good to confirm.)

Reply:
Good point, I think this does need to truncate, or each build will keep adding to the file (if an error occurs before it is removed).

Reply:
Sounds good (and safer anyways), updated the PR with a version that does truncation.

Reply:
OK, that doesn't work as-is, looking into it now.